### ISSUE TITLE
test-ibm-macos10.15-x64-1.nodejs.org

### ISSUE THREAD

**Issue by sstone1** [nodejs/build#3372]
👋 I used to work for IBM at Hursley, and at some point a long time ago I set up a Mac Mini for development on an IBM product.

It seems as if this Mac Mini is still alive and now named `test-ibm-macos10.15-x64-1.nodejs.org`. It still looks like it's in Hursley somewhere. I know this because it shows up in my Find My. I can `Mark As Lost`, or I can `Erase This Device`, but I cannot remove it from my account. 

I think someone needs to turn off Find My on the host to unregister it. Can someone look at this please?

@sxa @richardlau maybe one for you? I've emailed a few folks at IBM and they think given the name it may have moved to Red Hat.

**Comment by richardlau** [comment]
If it's in Hursley it probably didn't move with us to Red Hat. It sounds like it was one of the machines we used to use for the IBM SDK for Node.js but I have no current knowledge of it.

**Comment by sxa** [comment]
@sstone1 To clarify, are you saying it was registered to your account and some geolocation is showing that it's in Hursley and active just now? As Richard says we are no longer based at Hursley so it's not a machine either of us would be able to do anything with.

**Comment by sstone1** [comment]
@sxa correct, I had logged into iCloud on it at some point and I must have enabled the "Find My Mac" feature. I can see it's online, I can see it's in Hursley, but the most I can figure out about it is that it's a Macmini6,2.

Thanks for getting back to me - I'll go back to the IBM folks and see what they want to do.

**Comment by sej-jackson** [comment]
Hi @sstone1 👋 

I can ping `test-ibm-macos10.15-x64-1.nodejs.org`, but that address is showing up as North Bergen, New Jersey (rather a long way from Hursley).
```
$ ping test-ibm-macos10.15-x64-1.nodejs.org
PING test-ibm-macos10.15-x64-1.nodejs.org (138.197.224.240): 56 data bytes
64 bytes from 138.197.224.240: icmp_seq=0 ttl=45 time=92 ms
```
I didn't think we had any macs left, but I'll ask around.

**Comment by richardlau** [comment]
@sej-jackson FYI 138.197.224.240 is Node.js' public web server (nodejs.org). Despite the name I don't believe the machine in question was ever connected to the public Node.js project and was instead used internally in IBM by the Runtimes team. 

We had a discussion among the Red Hatters who used to work in IBM and @BethGriggs thinks we may have transferred the machine to another IBM team (IIB?) prior to moving.

**Comment by sej-jackson** [comment]
Hi @richardlau 👋 

ok, so that would make it a red-herring and can't be used to track it down unfortunately.  

The one system that I do know being transferred to IIB was `jtcjs33` but that was AIX, and no longer exists now.  

Do you happen to remember any names of your macs?  I've left a message for Stephen H. also to find out what happened to his ones.

**Comment by richardlau** [comment]
Unfortunately I do not. `test-ibm-macos10.15-x64-1.nodejs.org` is the sort of name that got generated when we ran Node.js' public Ansible playbook to set up internal machines for connection to the internal Jenkins instance we had.

**Comment by sstone1** [comment]
👋 hello Sej!

Just to close the loop - me and Graham E agreed that I would remotely lock the Mac Mini - it should be offline and unusable, with a message showing on the screen to contact Graham (who has the PIN) to unlock it. This is the last I saw before locking it:

![image](https://github.com/nodejs/build/assets/7118158/b5e9e28a-8a18-4356-9721-907057d7f7bd)

Now it's locked, I've been able to remove it from my Find My so I can't see it anymore.
